docs/components/power-sensor/_index.md

```diff
@@ -238,6 +238,58 @@ power, err := myPowerSensor.Power(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetReadings
+
+Get the measurements or readings that this power sensor provides.
+If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional\[Dict\[str, Any\]\])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(Mapping\[str, Any\])](https://docs.python.org/3/library/typing.html#typing.Mapping): The measurements or readings that this sensor provides.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/client/index.html#viam.components.power_sensor.client.PowerSensorClient.get_readings).
+
+```python
+my_sensor = Sensor.from_robot(robot=robot, name='my_sensor')
+
+# Get the readings provided by the sensor.
+readings = await my_sensor.get_readings()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(map\[string\]interface{})](https://go.dev/blog/maps): The measurements or readings that this sensor provides.
+- [(error)](https://pkg.go.dev/builtin#error): Report any errors that might occur during operation.
+  For a successful operation, `error` returns `nil`.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/powersensor#Readings).
+
+```go
+myPowerSensor, err := powersensor.FromRobot(robot, "my_power_sensor")
```

Comment:
@martha-johnston Is this correct? From the Go SDK docs it looks like it takes in the power sensor as a parameter (https://pkg.go.dev/go.viam.com/rdk/components/powersensor#Readings), but I can see that is not how it was implemented in the client method? (https://github.com/viamrobotics/rdk/blob/5b41b50663f8b0aafdce80f61e142dfc394c7443/components/powersensor/client.go) Slightly confused

Reply:
I'm a bit confused by the question... what takes in a power sensor? from robot? that should be correct

Reply:
@sguequierre that's a helper function that lives solely in golang - not the API, it's a pattern followed by movement sensor and power sensor, @martha-johnston maybe we could follow up to rename those helpers so they're not as confusing in the go docs for the golang SDK. SOmething like `powerSensor.DefaultReadings` or something to differentiate the helper function from the API.

Reply:
Ok, thank you guys, but sorry, I am still confused. Is the way I am documenting this currently incorrect or correct? 

To rephrase in case I am being extra confusing- The helper function `Readings` is different from `Readings` in the API, but I want to document the Power Sensor API client version of Readings. Is that correct with what I have now?

Reply:
Yes - it is what you're doing now and what is correct - that helper function should be renamed. It won't show up in the golang doc string because the function `Readings` that is used in the API is the nested function in the struct `sensor.Sensor`, and once the new release is cut will be the function nested as `resource.Sensor` in the power sensor struct.

What you're doing is what we want to document and the correct wrapper signature.

Reply:
Ok, thank you!

Reply:
Sorry about the confusion! I'll add a ticket to create a helper to clarify that, but yes what you have now _is_ correct